dropwizard-assets/src/main/java/io/dropwizard/assets/AssetsBundle.java

```diff
@@ -91,8 +89,14 @@ public AssetsBundle(String resourcePath, String uriPath, String indexFile) {
      * @param assetsName          the name of servlet mapping used for this assets bundle
      */
     public AssetsBundle(String resourcePath, String uriPath, String indexFile, String assetsName) {
-        checkArgument(resourcePath.startsWith("/"), "%s is not an absolute path", resourcePath);
-        checkArgument(!"/".equals(resourcePath), "%s is the classpath root", resourcePath);
+        if (!resourcePath.startsWith("/")) {
```

Comment:
Should we check if resourcePath is null here too?

Reply:
It wasn't checked before. 🤷‍♂️

Reply:
The first removed line is not null safe but the second is. So maybe the original intent was to be null safe? Also, it seems that this is no longer ensuring that the path is not the root. Is that the intention?

Reply:
@isaki-x I'd suggest checking the full diff of `AssetsBundle`. 😉

Reply:
@joschi /facepalm

My bad.